### ISSUE TITLE
Espanol modal pop-up Site Survey 

### ISSUE THREAD

**Issue by pageignited** [hackforla/food-oasis#965]
### Overview
We need understand if a Spanish website is needed by making a survey accessible to users

### Why
Spanish Angelinos are statistically likely to look for food assistance. To help them have a better access to organisations in their native language a Spanish website version would be a great help.

### What
We would build a pop-up survey to track the interest of Angelinos in a Spanish version

### What a success look likes
Our goal is to measure how many weekly users will click on this survey vs our total weekly users for x weeks/months 

### Actions

- [ ] Validate the pertinence of a this kind of survey
- [ ] Create wireframes
- [ ] Discuss the feasibility with Eng

<details>
<summary>Details</summary>
As a Spanish speaker food seeker, I want to cast my vote to have the site display in Spanish so that I can find resources in my native language. 

Acceptance Criteria

Desktop/Mobile
- On the (Homepage and organization listings) need to display “View site in Espanol” button. This button would display to any user. 
- Once the button is clicked it needs to pop up a modal window.
- Need to display this text to display in the modal "Estamos trabajando en una versión en español del sitio. Gracias por tu interés Sigue revisando." ("We are working on a Spanish version of the site. Thanks for your interest Keep checking")
- We need the user to be able to close the pop-up.
- We only want to display this button to the users that have visited the site but have not clicked on it.
- Need once a user clicks it, we need to track that user_id for that event attribute and then don't display that button again to that user on any page again.
- Need to capture results with “UTM” string for Google Analytics or an event with an attribute for Stormly.

See Wireframe
![Food oasis (1)](https://user-images.githubusercontent.com/22136920/115762722-9f4ceb80-a358-11eb-8372-ce9c01ffaaf7.png)

**Comment by ktjnyc** [comment]
@pageignited Hi Andrew, I switched this to be assigned to you since you have done all the work on it!

**Comment by fancyham** [comment]
I understand that the goal is to try to discover how many people are trying to read in Spanish, but this UX would just be frustrating to the user.  

They've clicked a button saying "See in Spanish" and then there's nothing actually in Spanish other than "Come back later".  That's bait-and-switch and would make us look untrustworthy.

I strongly suggest we not do this.

**Comment by ExperimentsInHonesty** [comment]
- @fancyham I 100% agree with your assessment above.
  - The solution here, should be to add google translate to the website and then count 
      - Number of people who use google translate on our site monthly
      - Number of languages
      - Number of use of each languages monthly
  - Then based on demonstrated need, we build true translated content if warranted.
- @Benbaillou please followup with John about feasibility and lift  of integrating Google Translate and how this issue relates https://github.com/hackforla/food-oasis/issues/12

**Comment by fancyham** [comment]
Zooming out a bit on this one — what question are we trying to answer with this survey?

If we're trying to find out if there's a need for a Spanish (and other) -language free food directory, I'd say we already know the answer.

Demographic and food-insecurity data, and [previous literature research we did for Los Angeles](https://docs.google.com/document/d/1SN1n2ZZKf8AOhkQUFlEICNkagRuazDyzEcRpxUT0d4w/edit) already points to 'yes'.  Russian and other languages are also needed.  Accessibility is also needed.

So, closing this issue out as we already know there's a need and we need to figure out how to do support other languages (ideally, easily).  

No need to create a survey that confirms what we already know (and a survey on our own site is probably a poor way of figuring it out). 

Will additional data showing need change our priorities?  Maybe yes, but this survey wouldn't be a good way to gather that data.

Let's find a cheap and fast way to do other languages and start with that instead.
